### PR TITLE
Workshop edits

### PR DESCRIPTION
**Pull request by profxj**
This is mainly a band-aid to get Kastr working with the new detector.

Needs a proper treatment once `ProcessImages` is re-vamped.

### CODE REVIEW

File: pypeit/calibrations.py [hunk 1]
```diff
@@ -541,6 +541,7 @@ def get_slits(self, redo=False, write_qa=True):
         self.tslits_dict, _ = self.traceSlits.master(prev_build=prev_build)
         if self.tslits_dict is None:
             # Build the trace image first
+            #import pdb; pdb.set_trace()
```
Comment (jhennawi) [inline pypeit/calibrations.py]: Can you remove these comment lines?

File: pypeit/spectrographs/spectrograph.py [hunk 2]
```diff
@@ -307,6 +307,8 @@ def get_datasec_img(self, filename, det=1, force=True):
                                           transpose=transpose, binning=binning_raw)
                 # Assign the amplifier
                 self.datasec_img[datasec] = i+1
+            #from pypeit import ginga
```
Comment (jhennawi) [inline pypeit/spectrographs/spectrograph.py]: Same here, please remove these debugging lines.

File: pypeit/spectrographs/shane_kast.py [hunk 3]
```diff
@@ -323,6 +259,8 @@ def __init__(self):
 
         # TODO: NEED TO CHECK ORIENTATION OF DATASEC AND OSCANSEC ARE
```
Comment (kbwestfall) [inline pypeit/spectrographs/shane_kast.py]: Have we successfully done that now?  Or should we leave this comment in?

**Comment by jhennawi** [comment]
Hi @kbwestfall, 

I agree with your strategy although I already merged PR#656 into develop. Anyway, I suggest we merge the current existing PRs, and then run the dev suite and tests again and then tag. 

Joe

File: pypeit/calibrations.py (hunk 1 above)
Comment (profxj) [reply pypeit/calibrations.py]: removed

File: pypeit/spectrographs/spectrograph.py (hunk 2 above)
Comment (profxj) [reply pypeit/spectrographs/spectrograph.py]: done

File: pypeit/spectrographs/shane_kast.py (hunk 3 above)
Comment (profxj) [reply pypeit/spectrographs/shane_kast.py]: it worked, so I think we are good.

Removed this TODO

**Comment by profxj** [comment]
merging.

**Pull request merged**